tiktok_reporter_analysis/classify_videos.py

```diff
@@ -44,87 +43,10 @@ def classify_videos(frames_folder, results_path):
         one_third = min_frame + (max_frame - min_frame) // 3
         two_thirds = min_frame + 2 * (max_frame - min_frame) // 3
         # Find the frames that are closest to one third and two thirds of the way between min and max
```

Comment:
If you want, you could use this approach in classify_reported.py and extract it to common.py.